tests/integration/drivers/QtTestDriverTest.cpp

```diff
@@ -0,0 +1,65 @@
+#include <QtTest>
+#include <cucumber-cpp/autodetect.hpp>
+
+#include "../../utils/DriverTestRunner.hpp"
+
+using namespace cucumber;
+
+THEN(SUCCEED_MATCHER) {
+    ScenarioScope<SomeContext> ctx;
+    QVERIFY(true);
+}
+
+THEN(FAIL_MATCHER) {
+    ScenarioScope<SomeContext> ctx;
+    QVERIFY(false);
+}
+
+THEN(PENDING_MATCHER_1) {
+    pending();
+}
+
+THEN(PENDING_MATCHER_2) {
+    pending(PENDING_DESCRIPTION);
+}
+
+using namespace cucumber::internal;
+
+class QtTestStepDouble : public QtTestStep {
+public:
+    QtTestStepDouble() : QtTestStep() {
+        testRun = false;
```

Comment:
Initialize this member var in the initializer list.

Reply:
done